### PR TITLE
Feature: add delete command

### PR DESCRIPTION
**Pull request by Zorin95670**

### CODE REVIEW

File: src/services/PluginUninstallator.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ * @typedef {import('../models/Plugin').Plugin} Plugin
+ */
+
+/**
+ * Execute npm install command and read the package.json file to check the dependencies.
+ *
+ * @param {Plugin} plugin - Plugin to install.
+ * @returns {Boolean} True if install command is done and the dependencies are updated,
+ * otherwise false.
+ */
```
Comment (aboubakar14) [inline src/services/PluginUninstallator.js:16]: uninstall instead of install

File: src/commands/Uninstall.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import chalk from 'chalk';
+import * as CommandLinePluginRetriever from '../services/CommandLinePluginRetriever.js';
+import * as PromptPluginRetriever from '../services/PromptPluginRetriever.js';
+import * as PluginUninstallator from '../services/PluginUninstallator.js';
+import { generatePluginsFile } from '../services/PluginConfiguration.js';
+
+/**
+ * @typedef {import('../models/Plugin').Plugin} Plugin
```
Comment (aboubakar14) [inline src/commands/Uninstall.js:8]: not used ?

File: src/commands/Uninstall.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+ */
+
+/**
+ * Install plugin.
```
Comment (aboubakar14) [inline src/commands/Uninstall.js:12]: uninstall

File: src/commands/Uninstall.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+ * @param {Object} program - Program to run command.
```
Comment (aboubakar14) [inline src/commands/Uninstall.js:13]: be more precise on what a program is

File: README.md [hunk 5]
```diff
@@ -73,27 +76,30 @@ The user can select one or more plugins to install from a list of official plugi
 
 Tasks performed during installation:
 - Install plugin(s) from npm.
```
Comment (aboubakar14) [inline README.md:78]: using npm

File: README.md [hunk 6]
```diff
@@ -73,27 +76,30 @@ The user can select one or more plugins to install from a list of official plugi
...
+### Uninstall
 
-A future command will be introduced to perform this action.
+```
+"plugin:uninstall": "leto-modelizer-plugin-cli uninstall",
+```
 
-However, here are the actions you need to take to delete the plugin:
+Tasks performed during installation:
+- Unnstall plugin(s) from npm.
```
Comment (aboubakar14) [inline README.md:96]: using npm

File: src/commands/Uninstall.js (hunk 2 above)
Comment (Zorin95670) [reply src/commands/Uninstall.js:8]: Used to indicate that Plugin refers to our custom plugin class for ESLint, not any other.

File: src/services/PluginUninstallator.js (hunk 1 above)
Comment (Zorin95670) [reply src/services/PluginUninstallator.js:16]: done

File: src/commands/Uninstall.js (hunk 3 above)
Comment (Zorin95670) [reply src/commands/Uninstall.js:12]: done

File: src/commands/Uninstall.js (hunk 4 above)
Comment (Zorin95670) [reply src/commands/Uninstall.js:13]: done

File: README.md (hunk 5 above)
Comment (Zorin95670) [reply README.md:78]: done

File: README.md (hunk 6 above)
Comment (Zorin95670) [reply README.md:96]: done

File: src/commands/Uninstall.js (hunk 2 above)
Comment (aboubakar14) [reply src/commands/Uninstall.js:8]: I know but the class Plugin is not used in the file

**Review by aboubakar14 — APPROVED** [review]

**Pull request merged**